debian/cloud-init.prerm

```diff
@@ -1,6 +0,0 @@
-#!/bin/sh
-
-set -e
-rm -f /etc/cron.d/cloudinit-updates
```

Comment:
This hasn't been used in a very long time.

Reply:
Agreed, pre version 0.7.7